### PR TITLE
Changes in front-end

### PR DESCRIPTION
**Pull request by HarshdipD**
### What are you trying to accomplish?
Adding footer, and suggesting a few changes on events page

### How are you accomplishing it?
CSS

### Is there anything reviewers should know?
Nope

- [x] Is it safe to rollback this change if anything goes wrong?

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -62,31 +62,37 @@
         <div class="col-3"></div>
         <div class="col-6">
           <br>
+          <!-- text-align instead of align because align is depracted and no longer works-->
```
Comment (EricPickup) [inline app/views/layouts/application.html.erb]: No need to add your comments in the code, simply describe it in a comment on your change in the PR (like I am right now) or explain it in the PR description.
```suggestion
```

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -57,36 +57,41 @@
...
 
-    <div class="container-fluid">
+    <div class="container-fluid" style="90vh">
       <div class="row">
         <div class="col-3"></div>
         <div class="col-6">
           <br>
           <% if flash[:error] %>
-            <div class="alert alert-danger" role="alert" align="center">
+            <div class="alert alert-danger" role="alert" text-align="center">
```
Comment (Scetch) [inline app/views/layouts/application.html.erb]: Shouldn't the text-align be inside a style tag for each of these?

**Comment by EricPickup** [comment]
Still broken:
image

Lets work on fixing your local machine first so you can test it yourself. Message me when you have time.

**Comment by HarshdipD** [comment]
This is it... I tried a new footer design that should look good. Take a look or I'll really have to fix my machine lol

**Comment by HarshdipD** [comment]
@Scetch Can you try this PR on your computer and post a screenshot of the footer?

**Comment by HarshdipD** [comment]
@EricPickup - Please look into this too. If it's not working we can close this PR and I'll work from scratch :)

**Pull request closed without merging**